packages/my-react-package/src/ContractWithWallet.tsx

```diff
@@ -31,6 +31,7 @@ class ContractWithWallet {
     wallet: MetaWallet;
     chain: string | null;
     toGasStation: string | null;
+    toTXBuilder: string | null;
```

Comment:
Why is X caps in TX?
Avoid abbreviations as far as possible please. Use copilot to reduce typing